CrashLoopBackOff if vxlan interface exists already

```

   DEBUG |               clabernetes | containerlab launched successfully
    INFO |               clabernetes | connectivity mode is 'vxlan', setting up any required tunnels...
   DEBUG |               clabernetes | resolved remote vxlan tunnel service address as '10.103.19.129'
   DEBUG |               clabernetes | using following args for vxlan tunnel creation (via containerlab) '[containerlab tools vxlan create --remote 10.103.19.129 --id 1 --link srl-e1-1 --port 14789]'
    INFO |               clabernetes |           
   ERROR  
          

    INFO |               clabernetes |   Error adding vxlan link host:vx-srl-e1-1: file exists.                                                              


    INFO |               clabernetes |   Try --help for usage.


   FATAL |               clabernetes | failed setting up tunnel to remote node 'srls' for local interface 'e1-1', error: exit status 1
panic:    FATAL |               clabernetes | failed setting up tunnel to remote node 'srls' for local interface 'e1-1', error: exit status 1

goroutine 1 [running]:
github.com/srl-labs/clabernetes/util.Panic({0xc00012c630, 0x8f})
	github.com/srl-labs/clabernetes/util/signals.go:69 +0xe6
github.com/srl-labs/clabernetes/logging.(*instance).Fatalf(0xc00013eec0, {0x2245daa?, 0x3?}, {0xc00030ddd0?, 0x4?, 0xc00012b4a0?})
	github.com/srl-labs/clabernetes/logging/instance.go:172 +0x185
github.com/srl-labs/clabernetes/launcher/connectivity.(*vxlanManager).Run(0xc0002d4800)
	github.com/srl-labs/clabernetes/launcher/connectivity/vxlan.go:43 +0x20f
github.com/srl-labs/clabernetes/launcher.(*clabernetes).connectivity(0xc00045e600)
	github.com/srl-labs/clabernetes/launcher/connectivity.go:33 +0x1d3
github.com/srl-labs/clabernetes/launcher.(*clabernetes).startup(0xc00045e600)
	github.com/srl-labs/clabernetes/launcher/clabernetes.go:114 +0xda
github.com/srl-labs/clabernetes/launcher.StartClabernetes()
	github.com/srl-labs/clabernetes/launcher/clabernetes.go:76 +0x4ba
github.com/srl-labs/clabernetes/cmd/clabernetes/cli.Entrypoint.func2(0xc00013edc0?)
	github.com/srl-labs/clabernetes/cmd/clabernetes/cli/entrypoint.go:67 +0xf
github.com/urfave/cli/v2.(*Command).Run(0xc000644840, 0xc00013edc0, {0xc0001ad490, 0x1, 0x1})
	github.com/urfave/cli/v2@v2.27.7/command.go:276 +0x7be
github.com/urfave/cli/v2.(*Command).Run(0xc000644c60, 0xc00013ec80, {0xc000112060, 0x2, 0x2})
	github.com/urfave/cli/v2@v2.27.7/command.go:269 +0xa45
github.com/urfave/cli/v2.(*App).RunContext(0xc0000e4200, {0x24e2a28, 0x373cba0}, {0xc000112060, 0x2, 0x2})
	github.com/urfave/cli/v2@v2.27.7/app.go:333 +0x5a5
github.com/urfave/cli/v2.(*App).Run(0xc000116060?, {0xc000112060?, 0x36635d8?, 0xc000002380?})
	github.com/urfave/cli/v2@v2.27.7/app.go:307 +0x2f
main.main()
	./main.go:10 +0x2d
```

This happens if the container restarts for some reason (without the pod being deleted). To fix this, I have to manually delete the pod so that it's recreated. 

Topology:

```
---
apiVersion: clabernetes.containerlab.dev/v1alpha1
kind: Topology
metadata:
  name: srl
  namespace: c9s-lab1
spec:
  naming: prefixed
  definition:
    containerlab: |-
          name: srl
          topology:
            nodes:
              srls:
                kind: nokia_srlinux
                image: ghcr.io/nokia/srlinux
              srl:
                kind: nokia_srlinux
                image: ghcr.io/nokia/srlinux
            links:
              - endpoints: ["srls:e1-1", "srl:e1-1"]
  deployment:
    launcherLogLevel: debug
```

Steps to reproduce:
1. Create the above topology.
2. Execute `pkill manager` inside the pod to force the pod to restart.
3. Observe the issue mentioned above.

--- a/launcher/connectivity/vxlan.go
+++ b/launcher/connectivity/vxlan.go
@@ -115,6 +115,7 @@ func (m *vxlanManager) runContainerlabVxlanToolsCreate(
 
 	vxlanInterfaceName := fmt.Sprintf("%s-%s", localNodeName, cntLink)
 	m.logger.Debugf("Attempting to delete existing vxlan interface '%s'", vxlanInterfaceName)
+
 	err = m.runContainerlabVxlanToolsDelete(m.ctx, localNodeName, cntLink)
 	if err != nil {
 		m.logger.Warnf(

